packages/core/docs/migrate/2.3.0-rc.1/integrators.md

```diff
@@ -0,0 +1,228 @@
+# Migration guide 2.3.0-rc.1 for Integrators
+
+## Introduction
+
+This is a migration guide meant to help Integrators make their integrations and plugins compatible with a new version 2.3.0-rc.1.
+
+It only contains code examples. For more information about the new version, please refer to the [Overview](./overview.md) page.
+
+## API middleware
+
+### Updating `api-client`
+As described on the [Overview](./overview.md) page, API middleware lives next to Nuxt.js. That's why from now on, `api-client` must generate two files:
```

Comment:
`lives next to` here means that it's separate. Also on `Overview` page we already have this sentence:

> It's an Express server that lives beside Nuxt.js. It can be run in parallel on the same server and separately on a different server, for better scalability.